sdk/managedserviceidentity/Azure.ResourceManager.ManagedServiceIdentities/api/Azure.ResourceManager.ManagedServiceIdentities.netstandard2.0.cs

```diff
@@ -121,8 +121,6 @@ public partial class UserAssignedIdentityResource : Azure.ResourceManager.ArmRes
         public virtual Azure.ResourceManager.ArmOperation Delete(Azure.WaitUntil waitUntil, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation> DeleteAsync(Azure.WaitUntil waitUntil, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.ResourceManager.ManagedServiceIdentities.UserAssignedIdentityResource> Get(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Pageable<Azure.ResourceManager.ManagedServiceIdentities.Models.IdentityAssociatedResourceData> GetAssociatedResources(string filter = null, string orderby = null, int? top = default(int?), int? skip = default(int?), string skiptoken = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Need using customization code to keep these 2 methods in the new version to avoid breaking change, even it have been deleted

Reply:
Refer this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Azure.ResourceManager.Compute/src/Customize/CloudServiceRoleInstanceResource.cs, just copy the code (just these 2 methods) from old version and mark them as Obsolete and EditorBrowsable

Reply:
added